CONFIG.md

```diff
@@ -63,24 +63,53 @@ path: general.server_round_robin
 default: false
 ```
 
-Whether to use round robin for server selection or not.
+Whether to use round-robin for server selection or not.
 
 ### server_tls
 ```
 path: general.server_tls
 default: false
 ```
 
-Whether to use TLS for server connections or not.
+Attempt to establish a TLS connection to the DB server.
+
+When enabling this feature, it is also important to configure the `general.verify_server_certificate` setting.
+
+If this is disabled, it is equivalent to `sslmode=disable`.
+
 
 ### verify_server_certificate
 ```
 path: general.verify_server_certificate
 default: false
+accepted values: true / false / "only-ca"
 ```
 
 Whether to verify server certificate or not.
 
+This feature only works when the `general.server_tls` setting is enabled.
+
+`false` - Server certificate validation is not performed at all. If the server
+allows to establish TLS connection, the connection will be established via
+https protocol, if not, it will be established via http. (similar to the
+`sslmode=prefer` mode)
+
+`true` - Connection to the DB server will be made with full server certificate
+verification. (similar to `sslmode=verify-full` mode)
+
+`"only-ca"` - Connection to the DB server will be made with full server
+certificate verification except for the hostname. (similar to `sslmode=verify-ca` mode)
+
+### trust_os_certificates
+```
+path: general.trust_os_certificates
```

Comment:
I think this should be default true. Also, this makes me think, do we need a setting for this at all? When would one not want to trust the OS certificates? The inclusion of webpki wasn't done intentionally, it was just part of all the examples for rustls. I think this is the desired way to use TLS certificates with pgcat, not with webpki. I think I would remove this option and load OS certificates by default.

Reply:
I was thinking the same thing. If you think so, I share your point of view.